.github/workflows/unit_tests.yaml

```diff
@@ -57,12 +40,13 @@ jobs:
         run: python3 -m unittest unit_test/**/*.py
       - name: Test Seqr Chart Installation
         run: |
-          # This hack lets us skip some expensive setup within the seqr startup
```

Comment:
Note an example init for this with the helm chart install is here: https://stackoverflow.com/questions/55499984/postgresql-in-helm-initdbscripts-parameter